src/discord/discord.service.ts

```diff
@@ -0,0 +1,50 @@
+
+interface DiscordData {
+    username: string;
+    amount: string;
+    pr: string;
+    date: string;
+    discordWh: string;
+    avatar: string
+}
+
+export default async function addBountyToDiscord({
+    username,
+    amount,
+    pr,
+    date,
+    discordWh,
+    avatar
+}: DiscordData) {
+    const body = {
+        content: 'Hello @everyone',
+        tts: false,
+        color: 'white',
+        avatar_url: "https://d2szwvl7yo497w.cloudfront.net/courseThumbnails/main.png",
+        embeds: [
+            {
+                title: `Congratulations to ${username} 🎉🎉`,
+                description: `**Username : ** ${username}\n**Amount : ** ${amount}\n**PR : ** ${pr}\n**Date:** ${date}`,
+                image: {
+                    url: avatar,
+                },
+            },
+        ],
+    };
+
+    try {
+        const res = await fetch(`${discordWh}`, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(body)
+        });
+
+        // const respData = await res.json();
```

Comment:
Sir, You can also remove this commented logs they are not necessary and if possible can more comments or the documentation link from discord where this part was or reference if take any in the comments so that it can help in the future enhancements